internal/generator/vector/input/viaq_test.go

```diff
@@ -90,10 +97,23 @@ var _ = Describe("inputs", func() {
 		Entry("with an application that specs specific namespaces and exclude namespaces", logging.InputSpec{
 			Name: "my-app",
 			Application: &logging.Application{
-				Namespaces:        []string{"test-ns-foo", "test-ns-bar"},
```

Comment:
Do we cover this usecase where  using deprecated namespaces field?  The variant here is would now include all containers from these NSs